### PR TITLE
bt-349: Fix step 1 design

### LINKED ISSUE #349: Step 1 doesn't correspond the design [BUG]
**Description:**
The layout of the Step 1 page does not correspond to the approved design

**Environment:**
Win 11, Chrome 116

**Steps to reproduce:**
1. Sign in to the app as a Talent
2. Go to Step 1
3. Fill in all fields with valid data as on the screenshot

**Actual Result:**
1. Layout doesn't correspond the Figma design

**Expected Result:**
1. There shouldn't be a space between fields name and the "*" symbol as on the design
2. Profile name field should have color fill when it filled with autocomplete(skip for now, hard to reproduce, low priority)
3. Job title and Current location fields has differnent padding on top and on bottom(4, 5px)
4. Experience field should have 5 and then 5+ years values. Should have 0,5 years(not 0,5 year) value. Shouldn't have "no" text at the bottom.
5. Employment type field chechboxes text should correspond the design(Part-time, Relocation to another country, Freelance(projects))
6. Briefly tell... field should use the same text font in the text box as in the whole app. 
7. Current Location field should have Location with a capital letter as on the design.

**Visual Proof (screenshots, videos, text):**

Image

Different paddings
Image

**Severity:**
Minor

### PR DESCRIPTION
**Pull request by ptarho**
Other fixes are already implemented in this PR
(https://bsa2023.slack.com/archives/C05NY1E7CBS/p1694424041354769?thread_ts=1694423038.951579&cid=C05NY1E7CBS)

### CODE REVIEW

File: frontend/src/bundles/common/components/form-label/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.label {
+    span:last-child {
```
Comment (nika-sirak) [inline frontend/src/bundles/common/components/form-label/styles.module.scss]: B10 - https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

File: frontend/src/bundles/common/components/form-label/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        visibility: hidden;
+    }
+
+    span:last-child::before {
```
Comment (nika-sirak) [inline frontend/src/bundles/common/components/form-label/styles.module.scss]: B10 - https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

File: frontend/src/bundles/common/components/form-label/styles.module.scss (hunk 1 above)
Comment (ptarho) [reply frontend/src/bundles/common/components/form-label/styles.module.scss]: fixed

File: frontend/src/bundles/common/components/form-label/styles.module.scss (hunk 2 above)
Comment (ptarho) [reply frontend/src/bundles/common/components/form-label/styles.module.scss]: fiexd

**Review by nikita-remeslov — APPROVED** [review]

**Review by savinaDaria — APPROVED** [review]

**Comment by savinaDaria** [comment]
conflicts

**Pull request merged**